apps/bmd/src/AppSetupErrors.test.tsx

```diff
@@ -264,38 +264,4 @@ describe('Displays setup warning messages and errors scrrens', () => {
     await wait(() => !queryByText(noPowerDetectedWarningText))
     getByText(insertCardScreenText)
   })
-
-  it('Cause hardware status polling to catch', async () => {
```

Comment:
not sure I understand why this test was removed

Reply:
In discussion, there were some states which could theoretically happen, but which never happened, so we decided to remove this code and related tests.